packages/genesys-spark-components/src/components/beta/gux-calendar-beta/components/single/gux-calendar.tsx

```diff
@@ -314,37 +324,22 @@ export class GuxCalendar {
             week =>
               (
                 <div class="gux-content-week">
-                  {week.dates.map(
-                    day =>
-                      (
-                        <div
-                          data-date={day.date.getTime()}
-                          onClick={() => this.onDateClick(day.date)}
-                          role="button"
+                  {week.dates.map(day => {
+                    const isoDateStr = asIsoDate(day.date);
+                    return (
+                      <slot key={isoDateStr} name={isoDateStr}>
+                        <gux-day
+                          day={isoDateStr}
```

Comment:
I am not sure if the `day` property will work for example if you have a `gux-day` component like
`<gux-day slot="2024-05-05" day="2024-05-10"></gux-day`  the calendar will be out of sync. I am not sure what the best way to handle that scenario would be. My suggestion would be either a single source of truth like keeping the `slot` attribute and somehow accessing the value in `gux-day` through an attribute on the parent maybe but I am not sure how that would work with how the calendar is rendered. My other sugguestion would be to just log a warning.